src/utils/scaffold.ts

```diff
@@ -47,14 +47,36 @@ export async function buildScaffold(projectDir: string, verbose: boolean) {
 
   await execa('npx', ['-p', '@nestjs/cli', 'nest', 'new', '.', '--skip-install', '--package-manager', 'npm'], options);
 
-  const pathToMainTs = path.resolve(projectDir, 'src', 'main.ts');
+  modifyMainTs(path.resolve(projectDir, 'src', 'main.ts'));
+  if (addCds) {
+    addCatalogueModule(path.resolve(projectDir, 'src', 'app.module.ts'));
+  }
+  cli.action.stop();
+}
+
+function modifyMainTs(pathToMainTs: string) {
   const mainTs = fs.readFileSync(pathToMainTs, { encoding: 'utf8' });
-  const modifiedMainTs = mainTs.replace('.listen(3000)', '.listen(process.env.PORT || 3000)');
+  const modifiedListen = '.listen(process.env.PORT || 3000)';
```

Comment:
Since we're replacing code here anyway, what do you think of changing the default port to 8080? You need to change it anyway if you want to deploy on CF. (I'm aware that this is out of scope for this PR)